pkg/cover/backend/elf.go

```diff
@@ -196,3 +176,43 @@ func elfReadModuleCoverPoints(target *targets.Target, module *Module, info *symb
 	}
 	return pcs, nil
 }
+
+func alignUp(addr, align uint64) uint64 {
```

Comment:
Move this below elfGetModuleOffset.
Go does not require throwing the most uninteresting details at the reader first ;)